### DISCUSSION TITLE
Is there API access built in?

### DISCUSSION THREAD

**Discussion by Ryang20718** [bigcapitalhq/bigcapital#184] [Q&A]
Is there API access built in?

**Comment by xprnio** [comment]
Yes, since the software is built as a client-server application the API does exist separately as well.
You can find the Postman collection here: https://www.postman.com/bigcapital/workspace/bigcapital-api

**Reply by queball1999** [reply]
The workspace seems to be missing substantial documentation. How do I obtain the token necessary to authenticate with the server?

**Reply by xprnio** [reply]
Great question - probably best to inspect the network, or looking at the source code.

[Keep in mind that the bulk of work on this project has been done by a single developer since 2023 (with another developer sharing the load in 2022).](https://github.com/bigcapitalhq/bigcapital/graphs/contributors). So - any help is probably much appreciated, but nothing should be expected. Keeping documentation up to date while maintaining and advancing an open-source project like this alone isn't exactly "easy work". That's just one of the pain points of open-source software - you need to be ready to dig deep into the code yourself, and deduce information from it.
